llms/llama/requirements.txt

```diff
@@ -1,3 +1,4 @@
 mlx>=0.0.6
 sentencepiece
 torch
+pandas
```

Comment:
I don't think it requires pandas..

Reply:
actually it requires `numpy`. let me change that.